django_extensions/management/commands/shell_plus.py

```diff
@@ -121,11 +121,21 @@ def get_notebook():
                 except ImportError:
                     return traceback.format_exc()
 
+            def install_kernel_spec(app, ipython_arguments):
+                """install an IPython >= 3.0 kernelspec that loads django extensions"""
+                ksm = app.kernel_spec_manager
+                ks = ksm.get_kernel_spec('python')
+                ks.argv.extend(ipython_arguments)
+                ks.display_name = 'Django Admin'
```

Comment:
it would also be possible to use the KernelSpec whitelist, to make this the only available kernel.